apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingService.kt

```diff
@@ -111,6 +116,18 @@ class RealRevurderingService(
         }
     }
 
+    override fun lagreRevurderingInfo(behandlingsId: UUID, info: RevurderingInfo, saksbehandler: String): Boolean {
```

Comment:
Kan denne splittes opp i to metoder, en som er for å sjekke om den kan lagres mens en som lagrer hvis den faktisk skal kunne lagres?

Reply:
Du tenker å hente ut sjekken i en egen privat metode?